### ISSUE TITLE
In the properties panel, show options as drop-down list; and convert keyframe options to clickable icons

### ISSUE THREAD

**Issue by raindropsfromsky** [OpenShot/openshot-qt#5437]
In the **Properties** panel, the values of all parameters are shown in an identical oblong shape.

While this may look nice, it does not give a clue whether a given parameter has optional values of a numeric value. (all look identical)

If we right-click, the parameters with optional values show the list, and the others do not respond.
![image](https://github.com/OpenShot/openshot-qt/assets/9047168/42b515f8-d8e0-4257-a2fe-1382ab010cc6)

Some show additional options in the context menu
![image](https://github.com/OpenShot/openshot-qt/assets/9047168/8b05af3f-a408-4f36-b76e-0b784b75e86c)

Some others do not show any context menu:
![image](https://github.com/OpenShot/openshot-qt/assets/9047168/2059ef74-10fb-42e8-999e-6e9dd296d1a3)

It would be far easier if we had a drop-down for all parameters that have optional values.

The **insert keyframe** and **remove keyframe** options should be converted to tiny icons next to the labels.
(some apps show this as a stopwatch icon.)

**Comment by Colorjet3** [comment]
Agreed.  This will be implemented as part of the new UI redesign phase.
